tests/foreman/api/test_computeresource_azurerm.py

```diff
@@ -31,6 +31,7 @@
 class TestAzureRMComputeResourceTestCase:
     """Tests for ``api/v2/compute_resources``"""
 
+    @pytest.mark.e2e
     @pytest.mark.upgrade
     @pytest.mark.tier1
     @pytest.mark.parametrize(
```

Comment:
`test_positive_azurerm_host_provisioned` is more an ideal End to End Test, As it is testing whole functionality with Compute Resource.

Reply:
Updated.